lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/Jordan_Eldridge/AnimalTest.java

```diff
@@ -1,40 +1,69 @@
-package com.codedifferently.lesson7.Jordan_Eldridge;
+package com.codedifferently.lesson7.jordan_eldridge;
 
 import static org.junit.jupiter.api.Assertions.*;
 
-import com.codedifferently.lesson7.Jordan_Eldridge.Animal.AnimalType;
+import com.codedifferently.lesson7.jordan_eldridge.Animal.AnimalType;
+import com.codedifferently.lesson7.jordan_eldridge.Animal.InvalidAgeException;
+import java.util.List;
 import org.junit.jupiter.api.Test;
 
 public class AnimalTest {
 
   @Test
-  public void testCanBreathe() {
+  public void testConstructor() {
     Animal animal = new Animal("Dog", true, 5, "Brown", "Bark", AnimalType.MAMMAL);
-    assertTrue(animal.isCanBreathe());
+    assertEquals("Dog", animal.getName());
+    assertTrue(animal.getCanBreathe());
+    assertEquals(5, animal.getAge());
+    assertEquals("Brown", animal.getColor());
+    assertEquals(AnimalType.MAMMAL, animal.getType());
   }
 
   @Test
-  public void testGetAge() {
-    Animal animal = new Animal("Dog", true, 5, "Brown", "Bark", AnimalType.MAMMAL);
-    assertEquals(5, animal.getAge());
+  public void testCanFly() {
+    Animal animal = new Animal("Owl", true, 1, "Brown", "whistles", AnimalType.BIRD);
+    assertEquals("This animal can fly!", animal.canFly());
   }
 
   @Test
-  public void testGetColor() {
-    Animal animal = new Animal("Dog", true, 5, "Brown", "Bark", AnimalType.MAMMAL);
-    assertEquals("Brown", animal.getColor());
+  public void testMakeSound() {
+    Animal animal = new Animal("lizard", true, 3, "Green", "Hissing", AnimalType.REPTILE);
+    animal.makeSound();
```

Comment:
In order for this to be a valid test, there needs to be a proper assertion that will fail if the code changes the expected behavior.

Reply:
gotcha think I made the fix.